ironic/drivers/modules/drac/raid.py

```diff
@@ -677,8 +714,9 @@ def _assign_disks_to_volume(logical_disks, physical_disks_by_type,
 
         for disks_count in range(min_disks, candidate_max_disks + 1):
             if ('number_of_physical_disks' in logical_disk
-                and logical_disk['number_of_physical_disks'] != disks_count):
-                    continue
+                    and logical_disk[
```

Comment:
Upstream reviewers will probably ask you to revert this change since it's outside the scope of this patch.

Reply:
Done